_extras/guide.md

```diff
@@ -38,6 +38,10 @@ Then when a learner messages in chat, the helper can find their repo and look at
 
 ## Episode: [Hosting Websites on GitHub](../github-pages/index.html)
 
+### Adding a theme through the GitHub Pages Settings tab
+If you're pushed for time and want to quickly introduce adding themes to your website without lots of additional context around Jekyll there is now a `Theme Chooser` section in the GitHub Pages Settings tab. This allows users to briefly browse supported Jekyll themes for GitHub Pages and select one for their site.
+It will add a `_config.yml` file to the repository containing a single line that sets the theme i.e. `theme: jekyll-theme-cayman`. This will be committed automatically to your default branch and rebuild your site with the new theme.
```

Comment:
Please add another sentence that warns Instructors that taking this route with the Theme Chooser is incompatible with content in the later episodes of the lesson, particularly from _Page Layouts_ onward.